datastore/additional_data/sources/org_types.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+
+class OrgTypesSource(object):
+    """ This adds a well educated guess for the type of organisation the funding organisation is"""
+
+    ADDITIONAL_DATA_KEY = "fundingOrgType"
+    REGEX = 0
+    VALUE = 1
+
```

Comment:
We might want to have a default value that's applied to all rows and overwritten by the rules below. It could be something like "Grantmaking foundation"